frontend/lib/tests/rtl-pal.tsx

```diff
@@ -54,22 +54,22 @@ export default class ReactTestingLibraryPal {
 
   /** Click anything with the given text and selector. */
   click(matcher: RegExp|string, selector: string) {
-    rt.fireEvent.click(this.getByTextAndSelector(matcher, selector));
+    return rt.fireEvent.click(this.getByTextAndSelector(matcher, selector));
```

Comment:
RTL's `fireEvent` actually returns the result of the underlying [`dispatchEvent()`](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/dispatchEvent) call, which is useful to detect whether a handler called `event.preventDefault()`.  We need this for our tests, so I exposed the return value on `ReactTestingLibraryPal`'s relevant methods.